### PR TITLE
Ax format: add the options alpha and grayscale

### PR DESCRIPTION
**Pull request by gbalan-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/UI/MicroUI/images.rst [hunk 1]
```diff
@@ -245,6 +245,28 @@ Here is the list of the compressed formats:
...
+.. _section_image_alpha:
+
+Alpha Format
+~~~~~~~~~~~~
+
+As described above, the formats ``A8``, ``A4``, ``A2``, ``A1`` and ``A8_RLE`` (also called *Picto*) only handle the opacity information.
+The source image can be transparent or not, colored or grayscaled.
+The alpha format provides two options to interpret the source image's pixels:
+
+* ``grayscale``: The source image is first `grayscaled`_ and then drawn over a white background. The black pixels are encoded as fully opaque pixels, the white pixels as encoded fully transparent pixels and gray pixels as transparent pixel (the blacker the pixel, the more opaque the encoded opacity).
```
Comment (seonmej) [inline ApplicationDeveloperGuide/UI/MicroUI/images.rst:257]:
```suggestion
* ``grayscale``: The source image is first `grayscaled`_ and then drawn over a white background.
The black pixels are encoded as fully opaque pixels, the white pixels as fully transparent pixels and gray pixels as transparent pixels (the blacker the pixel, the more opaque the encoded opacity).
```

File: ApplicationDeveloperGuide/UI/MicroUI/images.rst [hunk 2]
```diff
@@ -245,6 +245,28 @@ Here is the list of the compressed formats:
...
+* ``alpha``: Only the opacity component is encoded (the R-G-B components are ignored).
+* *no option*: same as ``grayscale`` (backward compatibility).
+
+::
+
+   image1:A8:grayscale
+   image2:A8_RLE:alpha
+   image3:A4
+
+.. note:: The MicroUI ResourceImage OutputFormat A8 encodes as the option ``alpha``.
```
Comment (seonmej) [inline ApplicationDeveloperGuide/UI/MicroUI/images.rst:267]:
```suggestion
.. note:: The MicroUI ResourceImage OutputFormat A8 encodes in the same way as the option ``alpha``.
```

File: VEEPortingGuide/uiImageRaw.rst [hunk 3]
```diff
@@ -157,38 +163,77 @@ Here are the conversions of 32-bit to each format:
                   ;
       }
 
--  A8: 8-bit format, only transparency is encoded.
-   ::
+-  A8: 8-bit format, only the source image's transparency is encoded (option ``alpha``), or the source image is pre-processed (option ``grayscale``); see :ref:`section_image_alpha`
```
Comment (seonmej) [inline VEEPortingGuide/uiImageRaw.rst:166]:
```suggestion
-  A8: 8-bit format, only the source image's transparency is encoded (option ``alpha``), or the source image is pre-processed (option ``grayscale``); see :ref:`section_image_alpha`.
```

File: ApplicationDeveloperGuide/UI/MicroUI/images.rst (hunk 1 above)
Comment (gbalan-microej) [reply ApplicationDeveloperGuide/UI/MicroUI/images.rst:257]: cannot add a new line: this is a bullet and a new line breaks the compilation

Comment (seonmej) [reply ApplicationDeveloperGuide/UI/MicroUI/images.rst]: ah sorry, I think that you need to align the first word `The` with the first word of the previous line.

File: ApplicationDeveloperGuide/UI/MicroUI/images.rst [hunk 4]
```diff
@@ -245,6 +245,28 @@ Here is the list of the compressed formats:
...
+.. _section_image_alpha:
+
+Alpha Format
+~~~~~~~~~~~~
+
+As described above, the formats ``A8``, ``A4``, ``A2``, ``A1`` and ``A8_RLE`` (also called *Picto*) only handle the opacity information.
+The source image can be transparent or not, colored or grayscaled.
+The alpha format provides two options to interpret the source image's pixels:
+
+* ``grayscale``: The source image is first `grayscaled`_ and then drawn over a white background. The black pixels are encoded as fully opaque pixels, the white pixels as fully transparent pixels and gray pixels as transparent pixels (the blacker the pixel, the more opaque the encoded opacity).
```
Comment (seonmej) [inline ApplicationDeveloperGuide/UI/MicroUI/images.rst:257]:
```suggestion
* ``grayscale``: The source image is first `grayscaled`_ and then drawn over a white background.
  The black pixels are encoded as fully opaque pixels, the white pixels as fully transparent pixels and gray pixels as transparent pixels (the blacker the pixel, the more opaque the encoded opacity).
```

File: ApplicationDeveloperGuide/UI/MicroUI/images.rst (hunk 1 above)
Comment (gbalan-microej) [reply ApplicationDeveloperGuide/UI/MicroUI/images.rst]: you're right! I didn't know this trick

**Review by seonmej — APPROVED** [review]

**Pull request merged**